infra/README.md

```diff
@@ -1,3 +1,62 @@
+# Multichain Infrastructure Overview
+
+## Environments:
+- Testnet (Production)
+- Dev (Development)
+
+## Deployment:
+
+### Development
+#### This environment has been automated for deployment, simply make a pull request with your changes to the `develop` branch, get it reviewed, and merge the PR.
+  - Deployment steps:
+    1. A merged PR triggers the following Github Actions Workflows:
+      - [multichain-dev.yml](../.github/workflows/multichain-dev.yml)
+      - [deploy-multichain-dev-contract.yml](../.github/workflows/deploy-multichain-dev-contract.yml)
+    2. These workflows deploy a new imaged based off of the github SHA tag to the 8 GCP vms and restart the VM
+    3. Then, the smart contract for the dev environment is reinitialized
```

Comment:
redeployed instead of reinitialized. Reinitialized meant something very different lol